.github/workflows/build-and-test.yaml

```
@@ -21,15 +21,25 @@ on:
   push:
     branches:
       - main
-    paths:
-      - '!**.md'
 
 jobs:
   build:
     runs-on: ubuntu-18.04
     timeout-minutes: 90
+    strategy:
+      matrix:
+        go-version: [1.14, 1.15]
+      fail-fast: false
```

```suggestion
      fail-fast: true
```

to avoid unnecessary waste of resources